apps/reference/docs/guides/realtime/quickstart.mdx

```diff
@@ -0,0 +1,277 @@
+
+<div class="video-container">
+  <iframe
+    src="https://www.youtube-nocookie.com/embed/BelYEMJ2N00"
+    frameBorder="1"
+    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+    allowFullScreen
+  ></iframe>
+</div>
+
+## Prerequisites
+
+- Install the `supabase-js` client:
+
+```bash
+npm install @supabase/supabase-js
+```
+
+- Get the public `anon` access token from your project's [API settings](https://app.supabase.com/project/_/settings/api).
+
+## Broadcast
+
+Broadcast allows a client to send messages and multiple clients to receive the messages.
+
+The broadcasted messages are ephemeral. They are not persisted to the database and are directly relayed through the Realtime servers. This is ideal for sending information like cursor positions where minimal latency is important, but persisting them is not.
+
+### Cursor Positions
+
+For this example, cursor positions will be randomly generated.
+
+You can start sending a client's cursor positions to other clients in channel `room1`:
+
+```js
+const { createClient } = require('@supabase/supabase-js')
+
+const supabase = createClient(
+  'https://your-project-ref.supabase.co',
+  'anon-key',
+  {
+    realtime: {
+      params: {
+        eventsPerSecond: 10,
+      },
+    },
+  }
+)
+
+// Channel name can be any string.
+// Create channels with the same name for both the broadcasting and receiving clients.
+const channel = supabase.channel('room1')
+
+// Subscribe registers your client with the server
+channel.subscribe((status) => {
+  if (status === 'SUBSCRIBED') {
+    // now you can start broadcasting cursor positions
+    setInterval(() => {
+      channel.send({
+        type: 'broadcast',
+        event: 'cursor-pos',
+        payload: { x: Math.random(), y: Math.random() },
+      })
+      console.log(status)
+    }, 100)
+  }
+})
+```
+
+:::info
+JavaScript client has a default rate limit of 1 Realtime event every 100 milliseconds that's configured by `eventsPerSecond`.
+:::
+
+Another client can subscribe to channel `room1` and receive cursor positions:
+
+```js
+// Supabase client setup
+
+// Listen to broadcast messages.
+supabase
+  .channel('room1')
+  .on('broadcast', { event: 'cursor-pos' }, (payload) => console.log(payload))
+  .subscribe((status) => {
+    if (status === 'SUBSCRIBED') {
+      // your callback function will now be called with the messages broadcast by the other client
+    }
+  })
+```
+
+:::info
+`type` must be `broadcast` and the `event` must match for clients subscribed to the channel.
+:::
+
+### Roundtrip Latency
+
+You can configure the channel so that the server must return acknowledgement that it received the `broadcast` message. This is useful if you want to measure roundtrip latency:
```

Comment:
```suggestion
## Roundtrip Latency

You can also configure the channel so that the server must return an acknowledgement that it received the `broadcast` message. This is useful if you want to measure the roundtrip latency:
```